config/initializers/bootstrap_errors.rb

```diff
@@ -0,0 +1,30 @@
+ActionView::Base.field_error_proc = Proc.new do |html_tag, instance|
+  html = %Q(<span class="field_with_errors">#{html_tag}</span>).html_safe
+
+  form_fields = [
+    'textarea',
+    'input',
+    'select'
+  ]
+
+  elements = Nokogiri::HTML::DocumentFragment.parse(html_tag).css form_fields.join(', ')
+  
+  elements.each do |e|
+    if form_fields.include? e.node_name
+      if instance.error_message.kind_of?(Array)
+        html = %Q(
+          <span class="field_with_errors" data-container="body" data-toggle="popover" data-placement="top" data-content="#{instance.error_message.uniq.join(', ')}">
```

Comment:
Line is too long. [164/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.